lighthouse-core/gather/connections/connection.js

```diff
@@ -86,6 +86,22 @@ class Connection {
     this._eventEmitter.on(eventName, cb);
   }
 
+  /**
+   * Unbind listeners for connection events.
+   * @param {'protocolevent'} eventName
+   * @param {function(LH.Protocol.RawEventMessage): void} cb
+   */
+  off(eventName, cb) {
+    if (eventName !== 'protocolevent') {
```

Comment:
Test case

Reply:
We don't have any tests at all for this base connection class or any of its extensions at the moment. Given that it's going away and it's an exact copy of existing code with a single method change, I didn't think it important to test. 

Maybe this is new information, maybe not, but has your mind changed? :)

Reply:
Makes sense, no test case is fine